frontend/src/app/shared/components/header-project-select/header-project-select.component.html

```diff
@@ -5,22 +5,26 @@
   alignment="bottom-start"
   class="op-project-list-modal"
 >
-  <button
```

Comment:
This change results in the project selector not being reachable via keyboard anymore as the anchor does not have an `href`. Semantically, `button` is also the correct tag..